src/main/java/io/apisense/embed/influx/configuration/InfluxConfigurationWriter.java

```diff
@@ -125,8 +119,13 @@ public InfluxConfigurationWriter build() {
             return new InfluxConfigurationWriter(configuration, dataPath, writer);
         }
 
-        public Builder setHttpPort(int httpPort) {
-            addSection(new HttpConfigurationSection(httpPort));
+        public Builder setHttp(int httpPort) {
```

Comment:
Could you add some Javadoc here to mention the default value of `auth`, among other? 
It would simplify the usage in IDEs.